### PR TITLE
Update tools.yaml

### PR DESCRIPTION
**Pull request by dsobral**
Tools necessary for the tutorial

### CODE REVIEW

File: Metagenomics/docker/tools.yaml [hunk 1]
```diff
@@ -39,10 +39,36 @@
...
-- name: ucsc_twobittofa
+- name: suite_mothur
   owner: iuc
-  tool_panel_section_label: "Convert Formats"
+  tool_panel_section_label: "16S-Metagenomics"
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
   install_resolver_dependencies: True
-  install_tool_dependencies: False
+  install_tool_dependencies: True
+- name: xy_plot
```
Comment (bgruening) [inline Metagenomics/docker/tools.yaml]: can we include line breaks between every tool? This makes them more readable.

File: Metagenomics/docker/tools.yaml [hunk 2]
```diff
@@ -39,10 +39,36 @@
...
+  owner: devteam
+  tool_panel_section_label: "Graph/Display Data"
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
```
Comment (bgruening) [inline Metagenomics/docker/tools.yaml]: There 3 lines are by default enabled and can be removed.
```
  tool_shed_url: https://toolshed.g2.bx.psu.edu/
   install_resolver_dependencies: True
   install_tool_dependencies: True
```

File: Metagenomics/docker/tools.yaml (hunk 1 above)
Comment (dsobral) [inline Metagenomics/docker/tools.yaml]: Sure! It's a good idea.

File: Metagenomics/docker/tools.yaml (hunk 2 above)
Comment (dsobral) [inline Metagenomics/docker/tools.yaml]: Thanks!

**Pull request merged**